src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -59,6 +60,13 @@ paths:
   /attribute/{namespace}/{attribute_name}/_meta:
     $ref: 'paths/attribute_namespace_attribute_name_meta.yaml'
 
+  /build:
+    $ref: 'paths/build.yaml'
+  /build/{project}:
+    $ref: 'paths/build_project.yaml'
+  /build/{project}/_result:
+    $ref: 'paths/build_project_result.yaml'
+
```

As we are now using `project_name` for the parameter, I propose to use the same for the paths.

```suggestion
  /build/{project_name}:
    $ref: 'paths/build_project_name.yaml'
  /build/{project_name}/_result:
    $ref: 'paths/build_project_name_result.yaml'

```

---

done